main.py

```diff
@@ -0,0 +1,82 @@
+# This is a sample Python script.
+
+# Press Shift+F10 to execute it or replace it with your code.
+# Press Double Shift to search everywhere for classes, files, tool windows, actions, and settings.
+
+
+# Use a breakpoint in the code line below to debug your script.
+
+
+# Press the green button in the gutter to run the script.
+
+
+# See PyCharm help at https://www.jetbrains.com/help/pycharm/
+def main():
+    first_board = [[], [], [], [], [], [], [], [], [], []]
+    for g in range(0, 10):
```

Comment:
creating a board should be it's own function sometihng like `first_board = create_board()`